test/helper.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const { Grape } = require('grenache-grape')
+const async = require('async')
```

Comment:
i would've refrained from async usage even in tests. One day we'll return here to remove this

Reply:
well it's still in maintenance and has wide support of promises